### ISSUE TITLE
Effect sizes lmer

### ISSUE THREAD

**Issue by clarmar301** [neuropsychology/psycho.R#108]
Hello,

I used the analyze function for my lmer model and the summary function reveals, for example, a medium effect (is it possible to get a number instead of the interpretation?) even if the p-value is not significant. How is this possible?

Thank you very much!

```r
I can paste some R code here. Ideally, a reproducible example.
```

**Comment by DominiqueMakowski** [comment]
hey @clarmar301 and welcome to github.

effect sizes (in this case corresponding to standardized coefficients) are indeed independent from significance. In short, frequentist significance tells you (((in theory, and to simplify))) the certainty with which the effect is different from 0 whereas the effect size is just a measure of the effects magnitude. An effect can be strong but with huge uncertainty (confidence interval covering 0). Hope it helps, cheers

**Comment by clarmar301** [comment]
Thank you very much! This was very helpful. have another off-topic question. 
I saw your paper "Phenomenal, bodily and brain correlates of fictional reappraisal as an implicit emotion regulation strategy" and I think you used the same effect size measure there. You made statements like "There is a probability of 81.33% that this effect size is medium and 18.67% that this effect size is large." Would you tell me how you calculate the probability/the percentage or a keyword I can search for?

**Comment by DominiqueMakowski** [comment]
Right. This is allowed by the Bayesian framework (you can find an introduction to Bayesian modelling [here](https://easystats.github.io/bayestestR/articles/bayestestR.html)). 

In short, the Bayesian framework allows you to obtain a distribution of possible effects, and in this case of standardized coefficients. Based on this distribution of values, you can then compute the proportion of values in each effect size "category" (e.g., 0.1 - 0.2, 0.2 - 0.4, 0.4 - 0.6 etc.). In the example above, it means that 81.33% of the values fell in the 0.2 - 0.4 range.
